src/test/java/teammates/test/cases/logic/TeamEvalResultTest.java

```diff
@@ -24,256 +24,268 @@ public static void setUp() {
         
         //only one person submitted
-        int[][] input7 = 
-            {{  25,  25,  75 }, 
-             { NSB, NSB, NSB },
-             { NSB, NSB, NSB }};
-        
-        int[][] expected7 = 
-            {{  60,  60, 180 }, 
-             { NSB, NSB, NSB },
-             { NSB, NSB, NSB },
-             
-             {  NA,  50, 150 }, 
-             {  NA,  NA,  NA },
-             {  NA,  NA,  NA },
-             
-             {  NA,  50, 150 },
-             
-             {  NA,  25,  75 }, 
-             {  NA,  50,  150 },
-             {  NA,  50,  150 }};
+        int[][] input7 = {
+                {  25,  25,  75 }, 
+                { NSB, NSB, NSB },
+                { NSB, NSB, NSB }
+        };
+
+        int[][] expected7 = {
+                {  60,  60, 180 }, 
+                { NSB, NSB, NSB },
+                { NSB, NSB, NSB },
+
+                {  NA,  50, 150 }, 
+                {  NA,  NA,  NA },
+                {  NA,  NA,  NA },
+
+                {  NA,  50, 150 },
+
+                {  NA,  25,  75 }, 
+                {  NA,  50,  150 },
+                {  NA,  50,  150 }
+        };
 
         verifyCalculatePoints(input7, expected7);
-        
+
         //only one person, submitted for self only
-        int[][] input8 = 
-            {{  25, NSU, NSU }, 
-             { NSB, NSB, NSB },
-             { NSB, NSB, NSB }};
-        
-        int[][] expected8 = 
-            {{ 100, NSU, NSU }, 
-             { NSB, NSB, NSB },
-             { NSB, NSB, NSB },
-             
-             {  NA,  NA,  NA }, 
-             {  NA,  NA,  NA },
-             {  NA,  NA,  NA },
-             
-             {  NA,  NA, NA },
-             
-             {  NA,  NA,  NA }, 
-             {  NA,  NA,  NA },
-             {  NA,  NA,  NA }};
+        int[][] input8 = {
+                {  25, NSU, NSU }, 
+                { NSB, NSB, NSB },
+                { NSB, NSB, NSB }
+        };
+
+        int[][] expected8 = {
+                { 100, NSU, NSU }, 
+                { NSB, NSB, NSB },
+                { NSB, NSB, NSB },
+
+                {  NA,  NA,  NA }, 
+                {  NA,  NA,  NA },
+                {  NA,  NA,  NA },
+
+                {  NA,  NA, NA },
+
+                {  NA,  NA,  NA }, 
+                {  NA,  NA,  NA },
+                {  NA,  NA,  NA }
+        };
         verifyCalculatePoints(input8, expected8);
         
         //two-person team
         //TODO: special treatment for two-person teams?
-        int[][] input9 = 
-            {{  50, 150 }, 
-             {  80, 80 }};
-        
-        int[][] expected9 = 
-            {{  50, 150 }, 
-             { 100, 100 },
-             
-             {  NA, 100 }, 
-             { 100,  NA },
-             
-             {  100,  100 },
-             
-             {  100,  100 }, 
-             {  80,  80 }};
+        int[][] input9 = {
+                {  50, 150 }, 
+                {  80, 80 }
```

Comment:
This will probably need to be edited too based on your format of whitespace padding is chosen :)